src/runtime/components/forms/FormGroup.vue

```diff
@@ -93,13 +97,17 @@ export default defineComponent({
 
     const size = computed(() => ui.value.size[props.size ?? appConfig.ui.input.default.size])
 
+    const labelFor = `label-for-${increment = increment < 1000000 ? increment + 1 : 0}`
```

Comment:
What about something like this to generate the random id? 
```ts
const seed = Math.random().toString(36).substring(7)
const labelFor = `${props.name}-${seed}`
```

Reply:
`const seed = Math.random().toString(36).substring(7)` can cause mismatched hydration as the server and client can have different values.

Reply:
@romhml, this is [my reference](https://github.com/vuejs/vue/issues/5886#issuecomment-308607131) for why I use an increment instead of a random value.

Reply:
@romhml, this is [my reference](https://github.com/vuejs/vue/issues/5886#issuecomment-308607131) for why I use an increment instead of a random value.

Reply:
Wouldn't it be better to add the `prop.name` inside the `labelFor`? `${props.name}-${increment...}`

Reply:
> Wouldn't it be better to add the `prop.name` inside the `labelFor`? `${props.name}-${increment...}`

okay

Reply:
@benjamincanac @romhml updated